Simulation Code/Base/Base.ino

```diff
@@ -106,7 +113,6 @@ void loop()
   unsigned long currentTime, elapsedTime, currentPumpTime, elapsedPumpTime; // Variables to store the current time and elapsed time
```

Comment:
Can these variables all be grouped with their respective struct? maybe we can simply create an instance in the loop if needed